packages/@ember/-internals/glimmer/index.ts

```diff
@@ -295,11 +295,8 @@
 export { templateFactory as template, templateCacheCounters } from '@glimmer/opcode-compiler';
 
 export { default as RootTemplate } from './lib/templates/root';
-export { default as Checkbox } from './lib/components/checkbox';
-export { default as TextField } from './lib/components/text-field';
-export { default as TextArea } from './lib/components/-textarea';
-export { default as LinkComponent } from './lib/components/-link-to';
 export { default as Input } from './lib/components/input';
+export { default as LinkTo } from './lib/components/link-to';
```

Comment:
New internal export used in tests that used to assert with the `LinkComponent`, but seem like they should test with this export now.